examples/gradle-kt-dsl/lib/src/test/kotlin/com/amazon/corretto/crypto/examples/AesGcm.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * This Kotlin source file was generated by the Gradle 'init' task.
+ */
+package com.amazon.corretto.crypto.examples
+
+import com.amazon.corretto.crypto.provider.AmazonCorrettoCryptoProvider
+import java.nio.charset.Charset
+import java.security.Security
+import javax.crypto.Cipher
+import javax.crypto.spec.GCMParameterSpec
+import javax.crypto.spec.SecretKeySpec
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+fun generateData(len: Int): ByteArray {
+    return ByteArray(len, Int::toByte)
+}
```

Comment:
Shouldn't this return random data rather than just all zeros?

Reply:
It's a good suggestion. This test is just exercising the AES-GCM mode and it's not much concerned with how IVs and keys are generated. I'll add another test that would exercise the LibCryptoRng backed by ACCP, once ACCP 2.0 is released.